tests/psi4numpy/rhf_hessian/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include(TestingMacros)
+
+add_regression_test(psi4numpy-rhf_hessian "psi;quicktests;smoketests;psi4numpy")
```

Comment:
Don't need both grad & hess in smoketests, as they're unlikely to break independently.

Reply:
Done!